polygon/sync/block_downloader.go

```diff
@@ -306,7 +306,7 @@ func (d *blockDownloader) downloadBlocksUsingWaypoints(
 			continue
 		}
 
-		d.logger.Debug(syncLogPrefix("fetched blocks"), "len", len(blocks), "duration", time.Since(batchFetchStartTime))
+		d.logger.Debug(syncLogPrefix("fetched blocks"), "start", blocks[0].NumberU64(), "end", blocks[len(blocks)-1].NumberU64(), "duration", time.Since(batchFetchStartTime))
```

Comment:
"len" is useful to easily see how many blocks were downloaded for the given "duration", we should return it back in addition to start and end

Reply:
I guess I'm assuming a certain level of mental arithmetic from the reader - if you shave start and end you also can know length.

For completeness I'll add it back at the end along with block/sec.